assets/src/skate.d.ts

```diff
@@ -3,3 +3,13 @@ export interface Route {
 }
 
 export type RouteId = string
+
+export interface Timepoint {
+  id: TimepointId
+}
+
+export type TimepointId = string
+
+export interface TimepointsByRoute {
+  [routeId: string]: Timepoint[]
```

Comment:
Can be the `RouteId` type instead of `string`.

Reply:
Typescript doesn't allow index signature parameter types to be type aliases.